### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by henry-richard**

### CODE REVIEW

**Comment by henry-richard** [comment]
leaving an issue comment 
Screenshot from 2022-09-26 21-24-27

**Comment by henry-richard** [comment]
**Slack user** please take a look

**Comment by wanda-carlson** [comment]
see changes requested

File: README.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 1111
 This is a public repository managed by PaidPaper. The code is based in Python. It has several GitHub actions.
```
Comment (wanda-carlson) [inline README.md:3]: comment 1 @henry-richard

File: README.md [hunk 2]
```diff
@@ -2,3 +2,4 @@
...
 test
```
Comment (wanda-carlson) [inline README.md:4]: comment 2

**Review by wanda-carlson — CHANGES_REQUESTED** [review]

**Comment by wanda-carlson** [comment]
adding suggestions

**Review by wanda-carlson — APPROVED** [review]

**Pull request merged**